encrypted-media/clearkey-mp4-unique-origin.html

```diff
@@ -0,0 +1,45 @@
+<!doctype html>
+<html>
+<head>
+    <meta charset=utf-8>
+    <meta name="timeout" content="long">
```

Comment:
My comment on the timeout meta tag applies to all the tests. Sorry - I did not make that clear.